commands/bot_test.go

```diff
@@ -50,3 +51,696 @@ func (s *MmctlUnitTestSuite) TestBotCreateCmd() {
 		s.Require().Contains(err.Error(), "could not create bot")
 	})
 }
+
+func (s *MmctlUnitTestSuite) TestBotUpdateCmd() {
+	s.Run("Should update a bot", func() {
+		botArg := "a-bot"
```

Comment:
Could you please add `printer.Clean()` to the first test case of each command as well? (for consistency)